presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSourceFactory.java

```diff
@@ -266,10 +277,19 @@ else if (column.getHiveColumnIndex() < fileColumns.size()) {
                     INITIAL_BATCH_SIZE,
                     exception -> handleException(orcDataSource.getId(), exception));
 
+            OrcDeletedRows deletedRows = new OrcDeletedRows(
+                    path.getName(),
+                    deleteDeltaLocations,
+                    new OrcDeletedDeltaPageSourceFactory(options, sessionUser, configuration, hdfsEnvironment, stats),
```

Comment:
Is it safe to pass all `options` there?
There're meant to control data reading, so may be not applicable to delta reading.

Reply:
Current options look safe to me. What do you think @dain

Reply:
I think it is fine for now.  If it is problematic, we can change it later.